macros.in

```diff
@@ -1120,6 +1122,23 @@ done \
 %{__patch} %{-p:-p%{-p*}} %{-q:-s}\
 %{__bzr} commit %{-q} -m %{-m*}
 
+# Subversion
+%__scm_setup_svn(q)\
+%{__svnadmin} create .svnrepos\
+%{__svn} mkdir %{-q} -m "Create directory structure." file://`pwd`/.svnrepos/trunk\
+%{__svn} checkout %{-q} file://`pwd`/.svnrepos/trunk ./\
+%{__svn} add %{-q} --force ./\
+%{__svn} commit %{-q} -m "Initial import." && %{__svn} update %{-q}
+
+# Subversion 1.6 doesn't have its own command to apply patches
+#%__scm_apply_svn(qp:m:)\
```

Comment:
Why is this still here, just commented out? Do you plan to fix it in a way that would let it pick the correct set of commands based on the SVN version or something?